scripts/merge-analyses.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
```

Comment:
Please add a mention and description of this script to analysis.md (to the "Analysis records are currently generated from" section).

Reply:
Done. I also filed https://bugzilla.mozilla.org/show_bug.cgi?id=1492322 to update the docs more generally, that's something I've been meaning to do.